src/lumigo_opentelemetry/dependencies/__init__.py

```diff
@@ -0,0 +1,72 @@
+# To provide better support and better data-driven product decisions
+# with respect to which packages to support next, the Lumigo
+# OpenTelemetry Distro for Python will report to Lumigo on startup the
+# packages and their versions used in this application, together with the
+# OpenTelemetry resource data to enable analytics in terms of which platforms
+# use which dependencies.
+#
+# This behavior is opt-out using the `LUMIGO_REPORT_DEPENDENCIES=false`
+# environment variable.
+
+from http.client import HTTPSConnection
+from json import dumps
+from os import environ
+from pkg_resources import Environment, get_distribution
+from typing import Any, Dict
+from urllib.parse import urlparse
+
+from opentelemetry.attributes import BoundedAttributes
+
+DEFAULT_CONNECTION_TIMEOUT = 3
+
+
+def report(url: str, lumigo_token: str, resource_attributes: BoundedAttributes) -> None:
+    dependencies = [
+        {
+            "name": distribution_name,
+            "version": get_distribution(distribution_name).version,
+        }
+        for distribution_name in Environment()
+    ]
+
+    data = dumps(
+        {
+            "resourceAttributes": _prepare_resource_attributes_for_marshalling(
+                resource_attributes
+            ),
+            "dependencies": dependencies,
+        }
+    )
+
+    _report_to_saas(url, lumigo_token, data)
```

Comment:
Can you add a log message saying that the dependencies were reported?

Reply:
No, it's noise in debugging

Reply:
Tracers are much harder to debug than services in the backend, therefore we try to add much more log messages than usual, it proved to be useful in the past.